Misc/NEWS.d/next/Library/2022-05-04-23-49-20.gh-issue-51574.8zvexM.rst

```diff
@@ -0,0 +1 @@
+The absolute path is returned, just like [mkdtemp()](https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp)
```

Comment:
Did you mean just like `mkstemp`? Also, if you say at the beginning what this is about (`tempfile.mkdtemp()`) it will read more clearly.